keanu-project/src/main/java/io/improbable/keanu/vertices/dbl/probabilistic/MultivariateGaussianVertex.java

```diff
@@ -54,19 +55,23 @@ public MultivariateGaussianVertex(DoubleVertex mu, DoubleVertex covariance) {
      * @param covariance the scale of the identity matrix
      */
     public MultivariateGaussianVertex(DoubleVertex mu, double covariance) {
-        this(mu, ConstantVertex.of(DoubleTensor.eye(mu.getShape()[0])).times(covariance));
+        this(mu, ConstantVertex.of(DoubleTensor.eye(mu.getShape()[0]).times(covariance)));
```

Comment:
what's the difference between multiplying the tensor or the vertex by `double covariance`?

Reply:
One is the constant vertex eye(dimension) times the constant vertex covariance and the other is the constant vertex of eye(dimensions) times covariance.